CHANGELOG.md

```diff
@@ -7,6 +7,7 @@
 - Add support for `limits.keepalive_timeout` configuration. ([#1645](https://github.com/getsentry/relay/pull/1645))
 - Add support for decaying functions in dynamic sampling rules. ([#1692](https://github.com/getsentry/relay/pull/1692))
 - Stop extracting duration metric for session payloads. ([#1739](https://github.com/getsentry/relay/pull/1739))
+- Add error and sample rate fields to the replay event parser. ([#1745](https://github.com/getsentry/relay/pull/1745))
```

Comment:
Only one thing, it looks like we had a release yesterday, so this record will have to be moved into new unreleased section again.